README.md

```diff
@@ -1 +1,172 @@
+
+
+</div>
+
+<b>Motivation </b>
+
+Developed by the [AI Verify Foundation](https://aiverifyfoundation.sg/), Project Moonshot is one of the first tools to bring [Benchmarking](#benchmarking-with-moonshot) and [Red-Teaming](#red-teaming-with-moonshot) together to help AI developers, compliance teams and AI system owners <b>evaluate LLMs and LLM applications</b>.
+This repository contains the test assests intended to work with the main Moonshot Python Library. You can add on additional assets to this folder to extend Project Moonshot's testing capabilities.
+
+Go to [Project Moonshot Python Library Repository](https://github.com/aiverify-foundation/moonshot).
+
+</br>
+
+## Table of Contents
+
+#### 🔗 For accessing AI systems:
+- [Connectors](#connectors) are APIs to AI systems to be tested and AI systems that enable scoring metrics, attack modules and context strategies.
+- <b>Connector Endpoints</b> are ready-to-use connectors that have been configured with the necessary API tokens and parameters.
+
+#### 📊 For Benchmarking:
+- <b>Datasets</b> are a collection of input-target pairs, where the 'input' is a prompt provided to the AI system being tested, and the 'target' is the correct response (if any). 
+- <b>Metrics</b> are predefined criteria used to evaluate the LLM’s outputs against the targets defined in the recipe's dataset. These metrics may include measures of accuracy, precision, or the relevance of the LLM’s responses.
+- <b>Prompt Templates</b> are predefined text structures that guide the formatting and contextualisation of inputs in recipe datasets. Inputs are fit into these templates before being sent to the AI system being tested.
+- [Recipes](#recipes) are a benchmarks that are ready to be administered onto an AI system, consisting minimally of a dataset and a metric.
+- [Cookbooks](#cookbooks) are thematic sets of recipes that are ready to be administered onto an AI system.
+
+#### ☠️ For Red-Teaming:
+- [Attack Modules](#attack-modules) are techniques that will enable the automatic generation of adversarial prompts for automated Red-Teaming.
+- <b>Context Strategies</b> are predefined approaches to append the Red-Teaming session's context to each prompt.
+- <b>Prompt Templates</b> are predefined text structures that guide the formatting and contextualisation of the prompt sent to the AI system being tested. User-input prompts are fit into these templates before being sent to the AI system being tested.
+
+#### 💯 Results:
+- Generated Outputs
+- Results Modules
+
+#### 🤝 Enablers:
+- Database Modules
+- I/O Modules
+- Runner Modules
+
+</br>
+
+See [here]() for more details on Project Moonshot Assets
+
+</br></br>
+
+## Getting Started
+
+### ✅ Prerequisites
+1. [Python 3.11](https://www.python.org/downloads/) (We have yet to test on later releases)
+
+2. [Git](https://github.com/git-guides/install-git)
+
+3. [Moonshot Python Library](https://github.com/aiverify-foundation/moonshot)
```

Comment:
should we link this to the pypi instead?

Reply:
You mean Moonshot Python Library? I think don't need. But I think the phrase is too deliberate. Shall we just change to "Moonshot Library"?